varcode/util.py

```diff
@@ -28,7 +28,11 @@
 _transcript_ids_cache = {}
 
 
-def random_variants(count, ensembl_release=MAX_ENSEMBL_RELEASE):
+def random_variants(
```

Comment:
https://github.com/hammerlab/varcode/issues/48

Also, since I'm currently using the non-determinism of this test to find edge cases, also going to add a commandline script for that purpose: 

https://github.com/hammerlab/varcode/issues/49